optimum/commands/neuron/cache.py

```diff
@@ -165,9 +167,21 @@ class SynchronizeRepoCommand(BaseOptimumCLICommand):
     @staticmethod
     def parse_args(parser: "ArgumentParser"):
         parser.add_argument("--repo_id", type=str, default=None, help="The name of the repo to use as remote cache.")
+        parser.add_argument(
+            "--cache_dir", type=str, default=None, help="The cache directory that contains the compilation files"
+        )
 
+    @requires_torch_neuronx
     def run(self):
-        synchronize_hub_cache(self.args.repo_id)
+        from libneuronxla.neuron_cc_cache import CacheUrl
+
+        if self.args.cache_dir is not None:
+            if not Path(self.args.cache_dir).is_dir():
+                raise ValueError(f"The {self.args.cache_dir} directory does not exist.")
+            cache_url = CacheUrl(self.args.cache_dir, url_type="fs")
```

Comment:
I'd rather move this inside the synchronize_hub_cache method, to avoid importing CacheUrl here.